### DISCUSSION TITLE
Certain beatmap failing to import

### DISCUSSION THREAD

**Discussion by NagBoard** [ppy/osu#19474] [Q&A] [answered]
### Type

Game behaviour

### Bug description

**Watch video below**
Beatmap link: https://osu.ppy.sh/beatmapsets/1817933#osu/3729755

### Screenshots or videos

https://youtu.be/UNZGdAom6vI

### Version

2022.723.0-lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/9226595/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9226596/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9226597/updater.log)
[database.log](https://github.com/ppy/osu/files/9226598/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9226599/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9226600/network.log)

**Comment by frenzibyte** [comment]
The logs you attached don't show the beatmap import failures you witnessed, please reproduce again and attach logs.

**Reply by NagBoard** [reply]
Sorry for the inconvenience. Sending you brand new logs:

[updater.log](https://github.com/ppy/osu/files/9226792/updater.log)
[database.log](https://github.com/ppy/osu/files/9226793/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9226795/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9226798/network.log)
[performance.log](https://github.com/ppy/osu/files/9226800/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9226801/runtime.log)

**Comment by frenzibyte** [comment, marked as answer]
The beatmap you attached doesn't work on stable as well. Please double-check the difficulty name and avoid unicode characters in the filenames (I'm not sure how you even managed to make this happen, did you input the character in the "Romanised" field somehow?).

As for the export re-import issue, you're likely suffering from https://github.com/ppy/osu/issues/17258.
